src/main/java/com/comcast/drivethru/api/HTTPRequestManager.java

```diff
@@ -86,10 +88,11 @@
      * Private constructor used by the Builder subclass to create and initialize the HTTPResposneManager object.
      * @param builder Builder object that has all the needed parts to make a HTTP(S) request
      */
-    private HTTPRequestManager(Builder builder)
+    HTTPRequestManager(Builder builder)
```

Comment:
So I see now why you made this change, but it shouldn't be publicly available...make it protected